src/assets/Generator.Shared/RequestUriBuilderExtensions.cs

```diff
@@ -6,120 +6,137 @@
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, TimeSpan value, string format, bool escape = true)
         {
-            builder.AppendQuery(name, TypeFormatters.ToString(value, format), escape);
+            builder.AppendQuery(name, ConvertToString(value, format), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, double value, bool escape = true)
         {
-            builder.AppendQuery(name, value.ToString(TypeFormatters.DefaultNumberFormat, CultureInfo.InvariantCulture), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, decimal value, bool escape = true)
         {
-            builder.AppendQuery(name, value.ToString(TypeFormatters.DefaultNumberFormat, CultureInfo.InvariantCulture), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, int value, bool escape = true)
         {
-            builder.AppendQuery(name, value.ToString(TypeFormatters.DefaultNumberFormat, CultureInfo.InvariantCulture), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, long value, bool escape = true)
         {
-            builder.AppendQuery(name, value.ToString(TypeFormatters.DefaultNumberFormat, CultureInfo.InvariantCulture), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, TimeSpan value, bool escape = true)
         {
-            builder.AppendQuery(name, XmlConvert.ToString(value), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, byte[] value, string format, bool escape = true)
         {
-            builder.AppendQuery(name, TypeFormatters.ToString(value, format), escape);
+            builder.AppendQuery(name, ConvertToString(value, format), escape);
         }
 
         public static void AppendQuery(this RequestUriBuilder builder, string name, Guid value, bool escape = true)
         {
-            builder.AppendQuery(name, value.ToString(), escape);
+            builder.AppendQuery(name, ConvertToString(value), escape);
         }
 
-        public static void AppendQueryDelimited<T>(this RequestUriBuilder builder, string name, IEnumerable<T> value, string delimiter, bool escape = true)
+        public static void AppendQueryDelimited<T>(this RequestUriBuilder builder, string name, IEnumerable<T> value, string delimiter, bool escape = true) where T : notnull
```

Comment:
I will remove this notnull constraint.
But since our generated code does not have the nullable static check enabled, this will not cause any difference.